executor/slow_query.go

```diff
@@ -381,7 +385,7 @@ func (st *slowQueryTuple) setFieldValue(tz *time.Location, field, value string,
 }
 
 func (st *slowQueryTuple) convertToDatumRow() []types.Datum {
-	record := make([]types.Datum, 0, len(slowQueryCols))
+	record := make([]types.Datum, 0, 64)
```

Comment:
Why changed this to 64？

Reply:
Because I don't want to export the `slowQueryCols` from `infoschema`, it was weird.